### PR TITLE
Remove process_event_ee

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

This removes EE PLUGIN_SERVER_INGESTION code that is unusable since we completely moved ingestion to the plugin server. Postgres process_event will be removed in another PR.

## Checklist

- [x] Django backend tests

### CODE REVIEW

**Comment by macobo** [comment]
Does this break demo data generation for clickhouse?

File: posthog/api/capture.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
+import datetime as dt
```
Comment (mariusandra) [inline posthog/api/capture.py:1]: Why not keep the `from datetime import datetime` line? Avoids using `dt.datetime` below...

**Review by mariusandra — COMMENTED** [review]
LGTM, though there's a merge conflict... and I don't know the answer to @macobo 's question about clickhouse demo data generation...

**Comment by Twixes** [comment]
Demo data gen uses `create_event` directly, bypassing `process_event_ee`.

**Review by mariusandra — APPROVED** [review]
image

**Pull request merged**